tests/tests_pytorch/loops/test_training_loop.py

```diff
@@ -140,6 +141,7 @@ def validation_step(self, *args):
     assert model.validation_called_at == (0, 5)
 
 
+@RunIf(standalone=True)
 def test_fit_loop_done_log_messages(caplog):
```

Comment:
why is this being marked as standalone? Let's undo this.

Reply:
@justusschock :otter:

Reply:
this is because it would fail otherwise due to importing the checkpoints and thereby importing PL as standalone.

#15309 is supposed to fix this.

Reply:
Ok, let's land this PR https://github.com/Lightning-AI/lightning/pull/15309 and see if it fixes it!

Reply:
https://github.com/Lightning-AI/lightning/pull/15309 is merged, can we try again @Borda?

```suggestion
def test_fit_loop_done_log_messages(caplog):
```